reactive/kubernetes_master.py

```diff
@@ -1008,12 +1021,23 @@ def loadbalancer_kubeconfig():
     hosts = loadbalancer.get_addresses_ports()
     # if there is a hacluster relation, use that vip/dns for the kubeconfig
     hacluster_vip = get_hacluster_ip_or_hostname()
-    if hacluster_vip:
-        address = hacluster_vip
+    # if the user gave us IPs for the load balancer, assume they know
+    # what they are talking about and use that instead of our information.
+    forced_lb_ips = hookenv.config('loadbalancer-ips').split()
+    if forced_lb_ips:
+        address = forced_lb_ips[get_unit_number() % len(forced_lb_ips)]
     else:
-        # Get the public address of the first loadbalancer so
-        # users can access the cluster.
-        address = hosts[0].get('public-address')
+        if hacluster_vip:
+            address = hacluster_vip
+        else:
+            if len(hosts) < 1:
+                # oops...hopefully we just lost our relation.
+                # skip this for now.
+                return
```

Comment:
If the `loadbalancer.available` flag is set, we shouldn't ever encounter this case. If we do, that should be fixed in interface-public-address.

I'm OK with having a check here too just in case -- hook errors are nasty -- but please include a log message so we know it happened.

Reply:
I'm fine with taking this out of this PR and getting more information. I just saw an error here when I removed the relation and thought I would try and fix it. Want me to just toss it for now and add a bug?

Reply:
Yeah, that works for me :+1: